test/regression/gufi_find.expected

```diff
@@ -11,7 +11,7 @@ expression may consist of:
     mindepth mmin mtime name newer
     path printf readable regex samefile
     size true type uid user
-    writable
+    verbose writable
```

Comment:
Remove verbose from this list. II is not part of the list of filters provided by GNU find.